Bio/Graphics/GenomeDiagram/_Colors.py

```diff
@@ -122,7 +122,7 @@ def translate(self, color=None, colour=None):
         return color
 
     def read_colorscheme(self, filename):
-        """ read_colorscheme(self, filename)
+        r""" read_colorscheme(self, filename)
```

Comment:
Can you remove the leading space as well, i.e.

```
    def read_colorscheme(self, filename):
         """read_colorscheme(self, filename)
```

(That still isn't PEP257 ready but its closer)